### PR TITLE
Admin tools - localization

### PR DESCRIPTION
**Pull request by litvinets**
admin-tools tabs localization
admins component
statistics
history log
providers
users

### CODE REVIEW

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -424,15 +457,21 @@
...
     "REMOVE_TEACHER_DATA": "Remove data about the teacher",
     "REMOVE_DATA": "Remove data",
     "ADD": "Add",
     "EDIT_USER": "Edit the user",
     "DELETE_USER": "Delete the user",
     "CONFIRM": "Confirm",
-    "CHANGE": "Change"
+    "CHANGE": "Change",
+    "ADD_INFO": "Add information",
+    "BLOCK": "BLock",
```
Comment (koliapatskal) [inline src/assets/i18n/en.json:470]: Please make the second letter lowercase

File: src/assets/i18n/uk.json [hunk 2]
```diff
@@ -466,5 +505,27 @@
...
+    "FORMAT": "Формат",
+    "GENERATE_REPORT": "Згенерувати звіти",
+    "NO_DATA": "У вас ще немає згенерованих звітів.",
+    "NO_DATA_SUBHEADER":"Задайте критерії і натисніть кнопку “Згенерувати звіти”, щоб побачити їх тут",
+    "TITLE": "Назва",
+    "PERIOD": "Період",
+    "CREATE_DATE": "Дата створення",
+    "DOWNLOAD": "Завантажити",
+    "WORKSHOPS_DAILY": "За останні 24 години",
+    "WORKSHOPS_YEAR": "За рік"
```
Comment (koliapatskal) [inline src/assets/i18n/uk.json:529]: You forgot to add the translation WORKSHOPS_DAILY and WORKSHOPS_YEAR to the English version, please add

File: src/assets/i18n/en.json [hunk 3]
```diff
@@ -214,6 +215,24 @@
       "ACCEPTED": "Accapted",
```
Comment (koliapatskal) [inline src/assets/i18n/en.json:215]: Accapted -> Accepted

**Review by koliapatskal — CHANGES_REQUESTED** [review]
In the admin component, tabs and the add button are not localized

image

**Comment by litvinets** [comment]
@koliapatskal thanks for the comments! 
admins component is localized in the another PR

File: src/assets/i18n/uk.json [hunk 4]
```diff
@@ -214,6 +215,25 @@
...
     },
+    "EMAIL_CONFIRMATION_STATUSES" : {
+      "PENDING": "Очікує підтвердження ел. пошти",
+      "CONFIRMED": "Ел. пошта підтверджена"
+    },
+    "PROVIDER_STATUSES":{
+      "APPROVED": "Заклад підтверджено",
+      "PENDING": "Очікує підтвердження",
+      "EDITING": "Потребує редагування",
+      "BLOCK": "Заклад заблокований"
```
Comment (koliapatskal) [inline src/assets/i18n/uk.json:226]: Please, add BLOCK to en.json file

**Review by koliapatskal — APPROVED** [review]

**Pull request merged**